tokio-util/src/time/mod.rs

```diff
@@ -16,6 +16,36 @@ pub mod delay_queue;
 
 #[doc(inline)]
 pub use delay_queue::DelayQueue;
+use futures_core::Future;
+use tokio::time::Timeout;
+
```

Comment:
Please group these imports with the `use std::time::Duration` import. We usually separate re-exports from normal imports.

Reply:
Ah got it, done. Thanks for the reviews